ballot/models/referendum.py

```diff
@@ -19,7 +19,7 @@ class Referendum(BallotItem, SocialMediaMixin):
     # TODO: Set up a save() event, or overload save(), to auto-populate
     #   BallotItemSelections (YES/NO)
     name = models.CharField(
-        max_length=255, help_text='The referendum number or the name '
+        max_length=255, help_text='The descripteve referendum name or the name '
```

Comment:
I find this to also be confusing (and misspelled). How about `A short referendum title or description`? Should we change this to `title`?